doc/source/overview/algorithms.md

```diff
@@ -29,6 +29,8 @@ The following tables summarize the advised use cases for the current algorithms.
 |Detector|Tabular|Image|Time Series|Text|Categorical Features|Online|Feature Level|
 |:---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
 |[Kolmogorov-Smirnov](../cd/methods/ksdrift.ipynb)|✔|✔| |✔|✔| |✔|
+|[Cramér-von Mises](../cd/methods/cvmdrift.ipynb)|✔|✔| |✔|✔|✔|✔|
```

Comment:
CVM not applicable to categorical features (including text).
FET not applicable to images and text. Tbh I'm not sure about saying KS and CVM are either but I see that's the existing convention so maybe discuss with @avl.

Reply:
Yes I had the same thoughts regarding images and text. Good spot re CVM 👍🏻

Reply:
Yes it's tricky as there is a big difference to what is possible (just based on data types) and what is recommended. The latter also then strongly depends on the type of preprocessing. So it becomes a rather opinionated table. I would like to redo this table anyway later and separate things out such as online vs. offline / data modalities / insight at the feature and/or instance level.

Reply:
@arnaudvl related to this, we're now on the latest sphinx so we can redo this entire page with panels/tabs etc when we have time...